mtalg/random/__init__.py

```diff
@@ -3,17 +3,17 @@
 
 __RNG = MultithreadedRNG()
 
-get_methods = lambda obj: [m for m in dir(obj) if callable(getattr(obj, m)) and m[0]!='_']
+get_methods = lambda obj: [m for m in dir(obj) if callable(getattr(obj, m)) and m[0] != '_']
 
 for method in get_methods(__RNG):
     argsspec = inspect.getfullargspec(getattr(__RNG, method))
     args_all, defaults = argsspec.args[1:], argsspec.defaults or []
-    kwargs = {k:v for k,v in zip(args_all[len(args_all)-len(defaults):], defaults)}
-    args = args_all[:len(args_all)-len(defaults)]
+    kwargs = {k: v for k, v in zip(args_all[len(args_all) - len(defaults):], defaults)}
+    args = args_all[:len(args_all) - len(defaults)]
+
 
     def func(*args, **kwargs):
         getattr(__RNG, method)(*args, **kwargs)
-        return __RNG._values
```

Comment:
@WWakker all clear except this - I don’t get how this can be the same? It doesn’t matter if the method returns it, one would need at least a return at the beginning of line 16,no?
is it tested?

Reply:
Oops you're right. I changed it quite a bit now though, sorry to mess with your dynamic code but a lot of unwanted stuff was imported into the namespace of mtalg.random and the functions didn't have docstrings, so I decided to go the manual way. I also added a `mtalg.set_num_threads`, have a look whether you agree.